apps/client/src/common/components/view-params-editor/constants.ts

```diff
@@ -110,189 +126,210 @@ export const MINIMAL_TIMER_OPTIONS: ParamField[] = [
     title: 'Key Colour',
     description: 'Background colour in hexadecimal',
     type: 'string',
+    defaultValue: '101010',
   },
   {
     id: 'text',
     title: 'Text Colour',
     description: 'Text colour in hexadecimal',
     type: 'string',
+    defaultValue: 'fffff',
   },
   {
     id: 'textbg',
     title: 'Text Background',
     description: 'Colour of text background in hexadecimal',
     type: 'string',
+    defaultValue: '101010',
```

Comment:
the same here

Reply:
Resolved with https://github.com/cpvalente/ontime/pull/610/commits/837d6de03de7923b490683dd872e213425c186b9